Fix Github Actions error with `__version__` module attribute

Zest recommends making a `__version__` module attribute available for convenience, however, this leads to an error when importing AWSsert in tests through Github Actions:

```
test/test_s3.py:4: in <module>
    from awssert.fixture import awssert
awssert/__init__.py:3: in <module>
    __version__ = pkg_resources.get_distribution("awssert").version
/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/pkg_resources/__init__.py:480: in get_distribution
    dist = get_provider(dist)
/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/pkg_resources/__init__.py:356: in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/pkg_resources/__init__.py:899: in require
    needed = self.resolve(parse_requirements(requirements))
/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/pkg_resources/__init__.py:785: in resolve
    raise DistributionNotFound(req, requirers)
E   pkg_resources.DistributionNotFound: The 'awssert' distribution was not found and is required by the application
```
The `__version__` attribute is commented currently as a workaround

--- a/awssert/__init__.py
+++ b/awssert/__init__.py
@@ -1,6 +1,5 @@
-# import pkg_resources
+import pkg_resources
 
-# TODO: Fix pkg_resources.DistributionNotFound error when running through Github Actions
-# __version__ = pkg_resources.get_distribution("awssert").version
+__version__ = pkg_resources.get_distribution("awssert").version
 
 from awssert.hooks import *

--- a/awssert/__init__.py
+++ b/awssert/__init__.py
@@ -1,5 +1,8 @@
 import pkg_resources
 
-__version__ = pkg_resources.get_distribution("awssert").version
+try:
+    __version__ = pkg_resources.get_distribution("awssert").version
+except pkg_resources.DistributionNotFound:
+    pass
 
 from awssert.hooks import *

